assignment/scripts/array-practice.js

```diff
@@ -75,20 +79,38 @@ console.log('The animals are now', animalArray);
 
 // 4.d TODO: Remove the food at the beginning of your array & 
 //     log both the food removed and the updated array
+removedFood = favoriteFoods.shift();
+console.log('Removed the first food', removedFood);
+console.log('The favorite foods are now', favoriteFoods);
 
 // 4.e (STRETCH) TODO: Replace the second food in your array
 //      with another one of your favorite foods.
 //      Then log the updated array.
+favoriteFoods[1] = 'mac and cheese';
+console.log(favoriteFoods);
 
 // 4.f (STRETCH) TODO: Sort your favoriteFoods array
 //     in reverse alphabetical order. Log the array.
+favoriteFoods.sort();
+favoriteFoods.reverse();
+console.log(favoriteFoods);
 
 // 4.g (STRETCH) TODO: Convert your array to a string
 //     putting the word "and" between each item.
 //     eg "tacos and pizza and pasta". Log the string.
+let foodString = favoriteFoods.join(' and ');
+console.log(foodString);
+
+// foodString=favoriteFoods.toString();
+// console.log(foodString);
+// foodString.join('and'); doesn't work because it's not an array
+// console.log(foodString);
 
 // 4.h (STRETCH) TODO: Make a new array that combines 
 //     the favorite foods array with the animals array.
 //     Then log the new array.
 //     It should look something like:
-//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
\ No newline at end of file
+//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
+
+let foodsAndPets = favoriteFoods.concat(animalArray);
```

Comment:
Nice. Could also have used the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) for this.